airflow/bin/cli.py

```diff
@@ -1315,7 +1315,7 @@ class CLIFactory(object):
         'ignore_all_dependencies': Arg(
             ("-A", "--ignore_all_dependencies"),
             "Ignores all non-critical dependencies, including ignore_ti_state and "
-            "ignore_task_deps"
+            "ignore_task_deps",
```

Comment:
better to change this to action="store_true", then this issue would have been avoided. Bonus points for fixing the othoer cases.